services/QuillLMS/spec/services/clever_integration/classrooms/importer_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe CleverIntegration::Classrooms::Importer do
+  let(:data) do
+    {
+      clever_id: clever_id,
+      name: 'clever classroom',
+      grade: '1'
+    }
+  end
+
+  subject { described_class.new(data) }
+
+  context 'classroom exists with clever_id' do
+    let(:clever_id) { '123_abc' }
+
+    before { create(:classroom, clever_id: clever_id) }
+
+    it 'runs classroom updater' do
+      expect { subject.run }.to_not change(Classroom, :count)
```

Comment:
Philosophical question: do we also want to set expectations around calls?  We could actually `expect` a call to Updater.run or Creator.run depending on which case we're looking at, but is that too focused on implementation rather than outcome (which the current tests are checking for already)?

Either way, we should almost certainly test something other than count because this particular case passes if nothing happens.  So we aren't actually confirming that update runs, just that creator doesn't.

Reply:
That's a really good point.  I will restructure the expectations.